datastore/gcloud/aio/datastore/array.py

```diff
@@ -0,0 +1,43 @@
+try:
+    from collections.abc import Sequence
+except ImportError:
+    from collections import Sequence
+from typing import Any
+from typing import Dict
+from typing import Iterator
+from typing import List
+
+import gcloud.aio.datastore.value
```

Comment:
Lets go with `from gcloud.aio.datastore import value` here, please (see [my comment on the other PR](https://github.com/talkiq/gcloud-aio/pull/139#discussion_r380926182))